scss/components/_panel-secondary.scss

```diff
@@ -0,0 +1,59 @@
+.panel-secondary {
+  $panel-control-margin: 10px;
+
+  box-shadow: $box-shadow-block;
+
+  .panel-body {
+    padding: $case-panel-secondary-padding;
+  }
+
+  .panel-footer {
+    padding: $case-panel-secondary-padding-small $case-panel-secondary-padding;
+  }
+
+  .panel-heading {
+    background: $crm-white;
+    line-height: 1;
```

Comment:
`1em` or the unit being used here by default.

Reply:
`line-height` values can be unit-less, see [here](https://allthingssmitty.com/2017/01/30/nope-nope-nope-line-height-is-unitless/)